resources/js/dist/filters-component-min.js

```diff
@@ -1,2 +1,2 @@
t){var i=this.getSearchParams();s.default.PARAMETER_ATTRIBUTES in i||(i[s.default.PARAMETER_ATTRIBUTES]={});var n=i[s.default.PARAMETER_ATTRIBUTES];if("price"===e||"range-slider"===this.facet.findologicFilterType)n[e]={min:t.min,max:t.max};else if("single"===this.facet.select)if(n[e]&&Object.values(n[e]).includes(t))if("cat"===this.facet.id&&t.includes("_"))n[e]=[t.split("_")[0]];else{var a=Object.values(n[e]).indexOf(t);delete n[e][a]}else n[e]=[t];else if(e in n){var r=this.getKeyByValue(n[e],t);if(-1===r){var o=Object.keys(n[e]).length;n[e][o]=t}else delete n[e][r]}else n[e]=[t];i[s.default.PARAMETER_ATTRIBUTES]=n,delete i[s.default.PARAMETER_PAGE],document.location.search="?"+$.param(i)},isValueSelected:function(e,t){var i=this.getSearchParams();if(!(s.default.PARAMETER_ATTRIBUTES in i))return!1;var n=i[s.default.PARAMETER_ATTRIBUTES];return e in n&&("cat"!==e&&"single"===this.facet.select&&n[e]===t||("cat"===e?-1!==this.getKeyBySuffix(n[e],t):-1!==this.getKeyByValue(n[e],t)))},getSelectedFilters:function(){var e=[],t=this.getSearchParams();if(!(s.default.PARAMETER_ATTRIBUTES in t))return e;var i=t[s.default.PARAMETER_ATTRIBUTES];for(var n in i)if("wizard"!==n)if("price"===n||this.isRangeSliderFilter(i[n])){var r=this.getFacetIdInfoMap(),o=r[n]&&r[n].unit?" "+r[n].unit:"";e.push({id:n,name:i[n].min+o+" - "+i[n].max+o})}else if("object"!==a(i[n]))e.push({id:n,name:i[n]});else{var u=i[n];for(var l in u)e.push({id:n,name:u[l].replace(/_/g," > ")})}return e},isRangeSliderFilter:function(e){return void 0!==e.min&&void 0!==e.max},removeSelectedFilter:function(e,t){t=t.replace(" > ","_");var i=this.getSearchParams(),n=i[s.default.PARAMETER_ATTRIBUTES];if("object"!==a(n[e])||"price"===e||this.isRangeSliderFilter(n[e]))delete n[e];else{var r=n[e];for(var o in r)r[o]===t&&delete n[e][o]}i[s.default.PARAMETER_ATTRIBUTES]=n,delete i[s.default.PARAMETER_PAGE],document.location.search="?"+$.param(i)},getSelectedFilterValue:function(e){var t=this.getSearchParams();if(!(s.default.PARAMETER_ATTRIBUTES in t))return null;var i=t[s.default.PARAMETER_ATTRIBUTES];return e in i?i[e]:null},getUrlParamValue:function(e){var t=this.getSearchParams();return e in t?t[e]:null},setUrlParamValue:function(e,t){var i=this.getSearchParams();i[e]=t,document.location.search="?"+$.param(i)},setUrlParamValues:function(e){var t=this.getSearchParams();e.forEach(function(e){t[e.key]=e.value}),document.location.search="?"+$.param(t)},getKeyByValue:function(e,t){for(var i in e)if(e.hasOwnProperty(i)&&e[i]===t)return i;return-1},getKeyBySuffix:function(e,t){for(var i in e)if(e.hasOwnProperty(i)&&e[i].endsWith(t))return i;return-1},removeAllAttribsAndRefresh:function(){var e=this.getSearchParams();delete e[s.default.PARAMETER_PAGE],delete e[s.default.PARAMETER_ATTRIBUTES],document.location.search="?"+$.param(e)},getFacetIdInfoMap:function(){var e={};return this.$store.state.itemList.facets.forEach(function(t){e[t.id]=t}),e}}}},{"../constants":14}]},{},[8,10,11,12,13,6,7,5,9,1,3,4,2,15]);
```

Comment:
Have you explicitly rebuilt these files after reverting the last commit?

Reply:
I've executed the command again, no new changes detected. So I guess we are fine?!

<img width="840" alt="Screen Shot 2021-08-25 at 16 08 25" src="https://user-images.githubusercontent.com/511559/130806075-3be9f6e4-9588-4c61-9345-5b96024f841e.png">

Reply:
Looks fine!